sdk/maps/azure-maps-render/src/main/resources/copyrightcaption.json

```diff
@@ -0,0 +1,4 @@
+{
+    "formatVersion": "0.0.1",
+    "copyrightsCaption": "© 1992 - 2022 TomTom."
+}
```

Comment:
Why is this json file needed? If this is for testing, it should be in `src/test/resources` and not in `src/main/resources`.